sourcegraph/sourcegraph
# own: integrate signal configurations into resolvers and indexers

Integrates the new signal config system with recent-contributor indexer and resolvers. Does not integrate repo filters into recent-views indexer yet.

## Test plan

1. Open an own panel, observe signals
2. Shut off a signal in admin UI and refresh own panel
3. signal should be gone

![CleanShot 2023-05-18 at 15 44 03@2x](https://github.com/sourcegraph/sourcegraph/assets/5090588/6e73e436-c9a3-4496-be4a-d410d8d58dd4)

For background:
1. start Sourcegraph
2. enable indexers
3. observe recent-contributor indexer starting

For repo filtering:
1. disable the recent-contributor signal
2. add a job to own_background_jobs with job_type 1
3. observe the job isn't dequeued

Janitor is covered with comprehensive automation tests.
